libraries/AP_HAL_Linux/Storage.h

```diff
@@ -2,7 +2,8 @@
 #define __AP_HAL_LINUX_STORAGE_H__
 
 #if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ERLEBOARD || CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_PXF
-#define LINUX_STORAGE_USE_FRAM 1
+#define LINUX_STORAGE_USE_FRAM 0
```

Comment:
Thanks for pointing it out. It'll probably make sense to have it in a different PR. I'll look at it this weekend.